utils/objects.py

```diff
@@ -15,7 +15,7 @@ def __init__(self, events):
 
     def goodElectrons(self):
         # # Good Electrons
-        electronQualityCut = (self.electrons.pt > self.leptonPt) & (abs(self.electrons.eta) < self.etaCut) & (self.electrons.iso < 0.1)
+        electronQualityCut = (self.electrons.pt > self.leptonPt) & (abs(self.electrons.eta) < self.etaCut) & (self.electrons.iso < 0.1) # iso is commented out for cut-based study
```

Comment:
Can you remove these comments about the iso commented out? Did you mean to keep this?